spec/features/workflow/unschedule_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.feature "Unschedule an edition" do
+  scenario do
+    given_there_is_a_scheduled_document
+    when_i_visit_the_summary_page
+    and_i_click_stop_scheduled_publishing
+    then_the_document_is_no_longer_scheduled_and_the_status_reverted
+    and_the_scheduled_datetime_is_still_persisted
+  end
+
+  def given_there_is_a_scheduled_document
+    schedule = create(:scheduling)
+    @datetime = Time.current.tomorrow.at_noon
```

Comment:
We can get rid of `.at_noon` here.  `.tomorrow` should be enough to indicate it's in the future.

Reply:
The `at_noon` method is an arbitrary choice which is needed due to if we just did `Time.current.tomorrow` ruby saves the time in the `@datetime` variable to a higher precision then postgres does when the `scheduled_publishing_datetime` is persisted (factory is built).

This caused the test on line 39 to fail when the dates were compared between the record and the saved variable so specifying an exact time was needed.